Makefile

```diff
@@ -201,12 +201,29 @@ xgo:
 install-os-dependencies:
 	_assets/scripts/install_deps.sh
 
-setup: install-os-dependencies dep-install lint-install mock-install gen-install update-fleet-config ##@other Prepare project for first build
+setup: install-os-dependencies dep-install lint-install mock-install deploy-install gen-install update-fleet-config ##@other Prepare project for first build
 
 generate: ##@other Regenerate assets and other auto-generated stuff
 	go generate ./static ./static/migrations
 	$(shell cd ./services/shhext/chat && exec protoc --go_out=. ./*.proto)
 
+prepare-release:
+	rm -rf /tmp/release-$(release_tag)
+	mkdir -p /tmp/release-$(release_tag)
+	mv build/bin/statusgo-android-16.aar /tmp/release-$(release_tag)/status-go-android.aar
+	mv build/bin/statusgo-ios-9.3-framework/status-go-ios.zip /tmp/release-$(release_tag)/status-go-ios.zip
+	${MAKE} clean
+	zip -r /tmp/release-$(release_tag)/status-go-desktop.zip . -x *.git*
+
+clean-release:
+	rm -rf /tmp/release-$(release_tag)
+
+deploy-install:
+	go get -u github.com/c4milo/github-release
+
+deploy:
+	github-release status-im/status-go $(release_tag) $(release_branch) "" "$(release_directory)"
```

Comment:
So do I understand correctly that anyone with repo write permission can just type `make deploy` and create a new release on github?
I wonder if we need to add more checks/questions here to prevent cases where people/scripts run `make deploy` just to see how it works (or by mistake). To me, publishing the release is an important thing, that need to be fool-proof and has to follow strict checklist. (I would  actually love to have multisig-like confirmation from other devs here, haha)

Reply:
Oh and what do we need `release_branch` for?

Reply:
We can add validation, `release_branch` is needed for github so it sets the right branch in the `archive` file (which we don't use currently, but it's created automatically by github)

Reply:
This is a very good point. Something like:
```Markdown
deploy:
	@read -p "Are you sure you want to create a new GitHub release? (y/n): " REPLY; \
	if [ $$REPLY = "y" ]; then \
	    github-release status-im/status-go $(release_tag) $(release_branch) "" "$(release_directory)"; \
	else \
	    echo "Aborting." && exit 1; \
	fi
```
BTW, I'd call this target `release` rather than `deploy`.